.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,11 +23,11 @@ Fixes #
 I have:
 
 - [ ] Read and followed Crossplane's [contribution process].
-- [ ] Added or updated unit tests.
-- [ ] Added or updated e2e tests, if necessary. (check with reviewers/maintainers if you're unsure whether E2E tests are necessary for the change).
 - [ ] Run `make reviewable` to ensure this PR is ready for review.
-- [ ] Added `backport release-x.y` labels to auto-backport this PR, if necessary.
-- [ ] Opened a PR updating the [docs], if necessary.
+- [ ] Added or updated unit tests.
+- [ ] Added or updated e2e tests.
+- [ ] Opened a PR updating the [docs].
```

Comment:
@plumbis I feel like we seldom check this box. In practice it seems uncommon for someone to have a docs PR open and ready when they open the PR that adds/touches new functionality. Usually instead I see more of a "I'll need to document this soon..." approach.

I wonder whether we should embrace this and maybe update the checklist item to be "created a docs tracking issue for the changes this PR introduces" or similar. With the intention being that the person opening the tracking issue will do the documenting. What do you think?

Reply:
Done. I also updated the contributing guide to explicitly state that you're expected to own documenting your feature before the next release. i.e. Raising a tracking issue doesn't mean asking someone else to document your thing.